app/views/practices/overview.html.erb

```diff
@@ -50,6 +50,24 @@
                 <%= f.select(:initiating_facility, options_for_select(['-Select-'], {selected: '-Select-', disabled: '-Select-'}), {}, {id: 'editor_facility_select', class: 'width-full height-5 usa-select'}) %>
               </div>
             </div>
+            <p class="margin-top-3"><span class="text-bold">Practice date:</span>&nbsp;<span>Select the month and year when this practice was created.</span>
+            <div class="grid-row grid-gap-4">
+              <div>
+                <%= label_tag 'editor_date_intiated_month', 'Month', class: 'usa-label' %>
+                <%= select_month(@practice.date_initiated.present? ? date_get_month(@practice.date_initiated) : nil, {prompt: '-Select-', prefix: 'date_initiated'}, {id: 'editor_date_intiated_month', class: 'width-card-lg height-5 usa-select'}) %>
```

Comment:
⚠️ Not sure if we would have `nil` `date_initiated` values but wanted to put a guard here just in case... same for the `select_year` below.

Reply:
i appreciate this!
always good to do this if we're running methods this on things like dates or numbers that we expect to be there for the method :+1: